engine/src/test/java/com/datatorrent/stram/plan/logical/LogicalPlanConfigurationTest.java

```diff
@@ -600,6 +609,104 @@ public void populateDAG(DAG dag, Configuration conf)
   }
 
   @Test
+  @SuppressWarnings({"UnnecessaryBoxing", "AssertEqualsBetweenInconvertibleTypes"})
+  public void testModuleUnifierLevelAttributes()
+  {
+    class DummyOperator extends BaseOperator
+    {
+      int prop;
+
+      public transient DefaultInputPort<Integer> input = new DefaultInputPort<Integer>()
+      {
+        @Override
+        public void process(Integer tuple)
+        {
+          LOG.debug(tuple.intValue() + " processed");
+          output.emit(tuple);
+        }
+      };
+      public transient DefaultOutputPort<Integer> output = new DefaultOutputPort<Integer>();
+    }
+
+    class DummyOutputOperator extends BaseOperator
```

Comment:
Yes, It is sufficient. Initially had more logic for testing purpose hence had created new output operator. Changed the code to use only DummyOperator.